; Stubtest requires mypy to pass before running, which would currently fail
; on numerous stubs. To allow running stubtest independently, we ignore mypy
; errors here.

[mypy]
ignore_errors = True
